### DISCUSSION TITLE
Can't Calculate Grade or Submit Assignment (Mindtap) URGENT!

### DISCUSSION THREAD

**Discussion by JacksonAcademy** [community/community#116484] [Codespaces]
### Select Topic Area

Bug

### Body

When I click "calculate grade" or "submit" in my mindtap, it will not submit, instead loading over and over again.

**Comment by shaikhmubin02** [comment]
Clear Browser Cache.

**Reply by JacksonAcademy** [reply]
Tried that already, also disabled adblocker

**Comment by BraydenC555** [comment]
Any fixes found yet? The companion is so buggy.
